src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,9 +1,46 @@
-import React, { ComponentProps } from 'react';
+import React, { ComponentProps, useState } from 'react';
+import { PopperProps, usePopper } from 'react-popper';
+
+export const Tooltip = ({ children, ...rest }: ComponentProps<'div'>) => (
+  <div className='str-chat__tooltip' {...rest}>
+    {children}
+  </div>
+);
+
+export const PopperTooltip = <T extends HTMLElement>({
+  children,
+  offset = [0, 10],
+  referenceElement,
+  placement = 'top',
+  visible: visible = false,
+}: React.PropsWithChildren<{
+  referenceElement: T | null;
+  offset?: [number, number];
+  placement?: PopperProps<unknown>['placement'];
+  visible?: boolean;
+}>) => {
+  const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(null);
+  const { attributes, styles } = usePopper(referenceElement, popperElement, {
+    modifiers: [
+      {
+        name: 'offset',
+        options: {
+          offset,
+        },
+      },
+    ],
+    placement,
+  });
+
+  if (!visible) return null;
 
-export const Tooltip = (props: ComponentProps<'div'>) => {
-  const { children, ...rest } = props;
   return (
-    <div className='str-chat__tooltip' {...rest}>
+    <div
+      className='str-chat__tooltip str-chat__button-tooltip'
```

Comment:
Will this tooltip always wrap buttons? Asking because of the class `str-chat__button-tooltip`

Reply:
This class isn't even defined within our styling so I'll remove it completely, must've missed it.